### DISCUSSION TITLE
Pandoc conversion issue

### DISCUSSION THREAD

**Discussion by audreywarne** [thegetty/quire#147] [Get Help] [answered]
I ran into an error when I attempted to convert a file of docx into md:

Unknown option --markdown-headings.
Try pandoc --help for more information.

It turns out I was running an old version of Pandoc and needed to update. Posting in case anyone else runs into a similar issue!

**Comment by Erin-Cecele** [comment, marked as answer]
As per @audreywarne, the solution is to update to the latest version of [Pandoc](https://pandoc.org/installing.html) and proceed with the instructions found in the [Quire documentation.](https://quire.getty.edu/documentation/fundamentals/#microsoft-word-to-markdown-conversion)
